src/app/pipes/keysPipe.ts

```diff
@@ -1,11 +1,16 @@
 import { Pipe, PipeTransform } from '@angular/core';
-@Pipe({name: 'keys'})
+@Pipe({ name: 'keys' })
 export class KeysPipe implements PipeTransform {
-  transform(value: { [x: string]: any; }, args?: string[]): any {
+  transform(value: { [x: string]: any }, args?: string[]): any {
     const keys = [];
     for (const key in value) {
-      keys.push({key: key, value: value[key]});
+      keys.push({ key: key, value: value[key] });
     }
     return keys;
   }
 }
+
+/**
```

Comment:
I think we should remove this file. Not used anywhere. Please check and clean up